server/apiRoutes/event.js

```diff
@@ -92,30 +92,22 @@ router.put('/events/:id', async (req, res, next) => {
 });
 
 //get single event
-router.get('/events/:id', async (req, res, next) => {
-  try {
-    const event = await Event.findOne({ where: { id: req.params.id } });
-    res.send(event);
-  } catch (err) {
-    next(err);
-  }
+router.get('/:id', async(req, res, next) => {
+    try {
+        const event = await Event.findOne({ where: { id: req.params.id },
+            include: [{model: User}, {model: Group}]    
+        });
```

Comment:
I think we need to wary of what/how much information we are sending to the client. What Users are being sent to the client here, all of them for the event? What data is attached to the User being sent? Should we be picky?

Sorry... Thats a lot of questions, but I'm not sure of the answer at the moment. I'm guessing you only need usernames and a few other things, maybe consider using attributes so we are certain of exactly what you need on the client side.

Reply:
I agree, I just threw it all in there because I didn't know what data I was going to use. I just went in and made this change, and pulled only the attributes I used.

Reply:
I think we can keep it as is for now. But probably just name and id are all you need.